### ISSUE TITLE
Breadcrumb links not styled in AppBar

### ISSUE THREAD

**Issue by smoyergh** [skeletonlabs/skeleton#1376]
### Current Behavior

Breadcrumb links (i.e., crumbs formatted with an anchor tag) are not rendered as expected, though they do function.

The stackblitz example attached adds breadcrumbs to the AppBar headline slot to demo this issue, but it happens in other slots as well.

### Reproduction / Steps To Reproduce

https://stackblitz.com/edit/skeletonlabs-repl-azypfo?file=src/routes/+layout.svelte

**Comment by endigo9740** [comment]
Hey @smoyergh funny enough I fixed this issue earlier today. The culprit is right here:

https://github.com/skeletonlabs/skeleton/blob/dev/src/lib/styles/typography.css#L36

The use of `:not(.app-bar a)` is exempting anchor styles for the app bar. However, I adjusted this today per this PR:

https://github.com/skeletonlabs/skeleton/pull/1374

This is a work in progress and may alter how Typography styles work in Skeleton, but just know this is coming.
